crowdsec-docs/docs/contributing/code.md

```diff
@@ -1,11 +1,10 @@
 ---
-id: contributing_code
+id: contributing_crowdsec
 title: CrowdSec
 sidebar_position: 1
 ---
 
 # Contribute to CrowdSec
 
- - If you want to report a bug, you can use [the github bugtracker](https://github.com/crowdsecurity/crowdsec/issues)
- - If you want to suggest an improvement you can use either [the github bugtracker](https://github.com/crowdsecurity/crowdsec/issues) or the [CrowdSecurity discourse](http://discourse.crowdsec.net) if you want to discuss 
-
+- If you want to report a bug, you can use [the github bugtracker](https://github.com/crowdsecurity/crowdsec/issues)
+- If you want to suggest an improvement you can use either [the github bugtracker](https://github.com/crowdsecurity/crowdsec/issues) or the [CrowdSecurity discourse](http://discourse.crowdsec.net) if you want to discuss
```

Comment:
I think we can also mention discord here

Reply:
this one is WIP, @mmetc  is working on it